### PR TITLE
FIX: Docker usage documentation

### LINKED ISSUE #12: error when running  `edf2bidsphysio` via docker
Tried to run edf2bidsphysio via the docker image using the test data in the repo and I am getting the following.

```bash
docker run --rm -it \
  -v $PWD/bidsphysio.edf2bids/tests/data/:/data \
    cbinyu/bidsphysio \
      edf2bidsphysio --infile /data/sample.edf --bidsprefix sub-10

/opt/venv/bin/edf2bidsphysio: line 3: import: command not found
/opt/venv/bin/edf2bidsphysio: line 4: import: command not found
/opt/venv/bin/edf2bidsphysio: line 7: __requires__: command not found
/opt/venv/bin/edf2bidsphysio: line 9: try:: command not found
/opt/venv/bin/edf2bidsphysio: line 10: from: command not found
/opt/venv/bin/edf2bidsphysio: line 11: except: command not found
/opt/venv/bin/edf2bidsphysio: line 12: try:: command not found
/opt/venv/bin/edf2bidsphysio: line 13: from: command not found
/opt/venv/bin/edf2bidsphysio: line 14: except: command not found
/opt/venv/bin/edf2bidsphysio: line 15: from: command not found
/opt/venv/bin/edf2bidsphysio: edf2bidsphysio: line 18: syntax error near unexpected token `('
/opt/venv/bin/edf2bidsphysio: edf2bidsphysio: line 18: `def importlib_load_entry_point(spec, group, name):'
```

### PR DESCRIPTION
**Pull request by oesteban**
This PR resolves the problems of the Docker image by setting an entrypoint using the virtualenv's python binary.

Resolves: #12.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,8 +84,14 @@ mkdir /tmp/bidsphysio && \
 ## Docker usage
 After pulling the latest version (`docker pull cbinyu/bidsphysio`), start a Docker container with:
 ```
-docker run [docker options] cbinyu/bidsphysio
+docker run [docker options] -v <datapath_host>:/data -v <outpath_host>:/output --entrypoint=/opt/venv/bin/python cbinyu/bidsphysio /opt/venv/bin/edf2bidsphysio --infile /data/<filename>.edf --bidsprefix /output/sub-01
 ```
+
+For example, using the test dataset in this repository:
+```
+docker run --rm -it -v $PWD/bidsphysio.edf2bids/tests/data/:/data:/data -v $PWD/output:/output --entrypoint=/opt/venv/bin/python cbinyu/bidsphysio /opt/venv/bin/edf2bidsphysio --infile /data/sample.edf --bidsprefix /output/sub-01/func/sub-01
```
Comment (Remi-Gau) [inline README.md]:
```suggestion
docker run --rm -it -v $PWD/bidsphysio.edf2bids/tests/data/:/data -v $PWD/output:/output --entrypoint=/opt/venv/bin/python cbinyu/bidsphysio /opt/venv/bin/edf2bidsphysio --infile /data/sample.edf --bidsprefix /output/sub-01/func/sub-01
```

Comment (Remi-Gau) [reply README.md]: small typo but other than  that it works

**Comment by oesteban** [comment]
@pvelasco @chrysapa @cbinyu this simple improvement of the documentation makes the example of utilization work correctly.

**Comment by pvelasco** [comment]
Thank you, @oesteban and @Remi-Gau !

**Pull request merged**